### ISSUE TITLE
Implement CRON event handling

### ISSUE THREAD

**Issue by diminish7** [diminish7/rave#6]

**Comment by bil-bas** [comment]
If you haven't seen this, it could be relevant (and why it didn't work before and should now).
http://code.google.com/p/wave-robot-java-client/issues/detail?id=15#c1

**Comment by diminish7** [comment]
I did some testing with some other cron-based bots. None of them seem to be working. I am assuming that means that cron events aren't working from the server's perspective. I am marking this task as blocked until Google gets cron events working again.

**Comment by bil-bas** [comment]
Yeah, even as little as a month ago, I see the Google guy who wrote Stocky saying the cron was unstable so they disabled it until they could implement a new system. Oh well, not much we can do.

**Comment by diminish7** [comment]
Here's an open ticket for this for the google wave team: http://code.google.com/p/google-wave-resources/issues/detail?id=417
